### ISSUE TITLE
Test Want To Read 'Stats' Page Interactive Elements for Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4912]
# Page to Test:

Want To Read 'Stats' - https://openlibrary.org/people/bpmcneilly/books/want-to-read/stats

# Steps

Steps for this task are outlined in #4897
 
# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by Yashs911** [comment]
@bpmcneilly I would like to work on this issue.

**Comment by Yashs911** [comment]
@bpmcneilly No issues.
